src/ClientHandle.cpp

```diff
@@ -1924,7 +1924,7 @@ void cClientHandle::Tick(float a_Dt)
 		{
 			m_PingID++;
 			m_PingStartTime = std::chrono::steady_clock::now();
-			m_Protocol->SendKeepAlive(m_PingID);
+			m_Protocol->SendKeepAlive(static_cast<UInt32>(m_PingID));
```

Comment:
Or should `m_PingID` and `SendKeepAlive` both be changed to `Int32`? After all the protocol seems to be using signed 32-bit integers for the ID ... at least according to [this source](http://wiki.vg/Protocol#Keep_Alive), that is.